backend/internal/repositories/post_repository.go

```diff
@@ -50,6 +51,34 @@ func (p *PostRepository) PostNew(userID uint, title, body string) error {
 	return result.Error
 }
 
+func (p *PostRepository) Update(id uint, title, body string) error {
+	m := model.Post{}
+	if err := p.Conn.First(&m, id).Error; err != nil {
+		return err
+	}
+
+	m.Title = title
+	m.Body = body
+	m.UpdatedAt = time.Now()
+	if err := p.Conn.Save(&m).Error; err != nil {
+		return err
+	}
+
+	return nil
```

Comment:
ここはifで分岐する意味ないと思うんだけど、こっちのが読みやすい？

```suggestion
	result := p.Conn.Save(&m)
	return result.Error
```